### PR TITLE
Sprint a

### PR DESCRIPTION
**Pull request by jakeberg**

### CODE REVIEW

File: submissions/sprint_a/Pipfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[dev-packages]
+pytest = "*"
+
+[packages]
+flask = "*"
+python-dotenv = "*"
+
+[requires]
+python_version = "2.7"
```
Comment (infosmith) [inline submissions/sprint_a/Pipfile:14]: Flask supports python 2.7 and >= 3.4. I recommend using 3.6 and above.
http://flask.pocoo.org/docs/1.0/installation/

File: submissions/sprint_a/__init__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def configure_app():
+    import os
+    from dotenv import load_dotenv
+    from flask import Flask, jsonify
+
+    PROJECT_ROOT = os.path.dirname("./")
+    env_path = os.path.join(PROJECT_ROOT, ".env")
+    load_dotenv(dotenv_path=env_path, verbose=True, override=True)
```
Comment (infosmith) [inline submissions/sprint_a/__init__.py:8]: use os.path.dirname(os.path.abspath(\_\_file__)) instead. It's safer.

File: submissions/sprint_a/__init__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+def configure_app():
+    import os
+    from dotenv import load_dotenv
+    from flask import Flask, jsonify
```
Comment (infosmith) [inline submissions/sprint_a/__init__.py:4]: You don't need to import jsonify here. It's a flask module but not part of flask configuration. You import jsonify anywhere in an app and it's fine.

File: submissions/sprint_a/__init__.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+    PROJECT_ROOT = os.path.dirname("./")
+    env_path = os.path.join(PROJECT_ROOT, ".env")
+    load_dotenv(dotenv_path=env_path, verbose=True, override=True)
+    app = Flask(__name__)
+
+    return app, jsonify
```
Comment (infosmith) [inline submissions/sprint_a/__init__.py:11]: limit your returns to instances of app, db, etc...

File: submissions/sprint_a/app.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+from . import app, jsonify
+
+
+@app.route('/')
+def generate_epithets():
+
```
Comment (infosmith) [inline submissions/sprint_a/app.py:6]: On functions this small, docstrings really aren't necessary, but a docstring or remove the newline.

File: submissions/sprint_a/app.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    response = {"epithets": []}
+    return jsonify(response)
+
+
+@app.route('/vocabulary')
+def vocabulary():
+    response = {"vocabulary": {}}
+    return jsonify(response)
```
Comment (infosmith) [inline submissions/sprint_a/app.py:14]: You can save yourself a variable assignment and just return the dictionary using jsonify.

**Review by infosmith — CHANGES_REQUESTED** [review]
Please add docstrings to all of your files.

**Comment by itsthejoker** [comment]
Finally graded!

**Pull request closed without merging**